### PR TITLE
LNX-201 Add a system for adding agents

### PR DESCRIPTION
**Pull request by Krystian030**

### CODE REVIEW

File: lynx/common/player.py [hunk 1]
```diff
@@ -10,10 +13,68 @@ class Player(Serializable):
     player_resources: Dict[str, int] = field(default_factory=dict)
     drop_area: Vector = field(default_factory=Vector)
     agents: dict[str, Agent] = field(default_factory=dict)
+    max_agents: int = 1
```
Comment (filip-szweda) [inline lynx/common/player.py:16]:
```
max_agents: int = 1

BASE_COSTS = {
    "Wood": 6,
    "Stone": 6,
}
```
[Optional] Move these values to a config (just like [here](https://github.com/group-project-gut/lynx-scene-host-python/blob/master/config.ini)) as e.g. `START_MAX_AGENTS`, `WOOD_COST`, `STONE_COST`

File: lynx/common/player.py [hunk 2]
```diff
@@ -10,10 +13,68 @@ class Player(Serializable):
...
+
+    BASE_COSTS = {
+        "Wood": 6,
+        "Stone": 6,
+    }
+
+    @property
+    def price_multiplier(self) -> int:
```
Comment (filip-szweda) [inline lynx/common/player.py:24]: What does this function do? It multiplies a price. So:
```suggestion
    def multiply_price(self) -> int:
```

File: lynx/common/player.py [hunk 3]
```diff
@@ -10,10 +13,68 @@ class Player(Serializable):
...
+        return 2 ** (self.max_agents - 1)
+
+    def alive_agents_count(self) -> int:
```
Comment (filip-szweda) [inline lynx/common/player.py:27]:
```suggestion
    def get_alive_agents_count(self) -> int:
```

**Review by filip-szweda — CHANGES_REQUESTED** [review]

File: lynx/common/player.py (hunk 2 above)
Comment (Krystian030) [reply lynx/common/player.py:24]: Property nazwyamy rzeczownikami - to jest pole NIE metoda

Comment (filip-szweda) [reply lynx/common/player.py:24]: My bad, I didn't notice

**Review by filip-szweda — APPROVED** [review]

**Pull request merged**